flows/inputs/msg.go

```diff
@@ -17,7 +17,7 @@ const TypeMsg string = "msg"
 // MsgInput is a message which can be used as input
 type MsgInput struct {
 	baseInput
-	urn         urns.URN
+	urn         *flows.ContactURN
```

Comment:
so that the URN at @run.input.urn behaves the same way as other context URNs in expressions